Unify on build platform (#810)
slsa-framework/slsa: docs/spec/v1.0/verifying-artifacts.md

        platform itself, such as by a malicious insider. Instead, verifiers
        SHOULD carefully consider which build platforms are added to the roots
        of trust. For advice on establishing trust in build platforms, see
        [Verifying build platforms](verifying-systems).
-   [Threat "F"]: SLSA Build L2 covers tampering of the artifact or provenance
    after the build. This is accomplished by verifying the `subject` and

>     signature in the steps above.

Reviewer: Line 152 below: `system` should be `platform`

## Architecture options

System implementers decide which part(s) of the platform will verify provenance:
the package ecosystem at upload time, the consumers at download time, or via a
continuous monitoring system. Each option comes with its own set of
considerations, but all are valid. The options are not mutually exclusive, but

> at least one part of a SLSA-conformant platform SHOULD verify provenance.

Reviewer: "Platform" doesn't really work here. The intention was really the overall producer-to-consumer chain of events. Here's a possible rewording that avoids the problem:

> There are several options for where provenance verification can happen, and these options are not mutually exclusive:
> [...]
> considerations, but all are valid and at least one SHOULD be used.

During package upload, a package ecosystem can ensure that the artifact's
provenance matches the expected values for that package name's provenance before

> accepting it into the package registry.  If possible, platform implementers SHOULD
> prefer this option because doing so benefits all of the package ecosystem's
> clients.

Reviewer: similarly:

```suggestion
accepting it into the package registry.  This option is RECOMMENDED whenever
possible because doing so benefits all of the package ecosystem's clients.
```

Author: Thanks. Changed locally to be included in the next push.

Consumers can continuously poll a monitor to detect artifacts that
do not meet the monitor's expectations. Detecting artifacts that fail

> verification is of limited benefit unless a human or another part of the platform
> responds to the failed verification.

Reviewer: similarly:

```suggestion
verification is of limited benefit unless a human or automated system takes
action in response to the failed verification.
```

Author: Thanks. Changed locally to be included in the next push.
